src/RemoteSymbolProvider/StadiaSymbolStoreSymbolProvider.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "RemoteSymbolProvider/StadiaSymbolStoreSymbolProvider.h"
+
+#include <absl/strings/str_format.h>
+#include <absl/strings/str_replace.h>
+
+#include "OrbitBase/File.h"
+#include "OrbitBase/FutureHelpers.h"
+#include "OrbitGgp/SymbolDownloadInfo.h"
+
+using orbit_base::CanceledOr;
+using orbit_base::Future;
+
+namespace orbit_remote_symbol_provider {
+ErrorMessageOr<std::unique_ptr<StadiaSymbolStoreSymbolProvider>>
+StadiaSymbolStoreSymbolProvider::Create(std::filesystem::path cache_directory,
+                                        orbit_http::DownloadManagerInterface* download_manager,
+                                        orbit_ggp::Client* ggp_client) {
+  std::string error_message = "Create StadiaSymbolStoreSymbolProvider failed with error:\n";
+  if (auto exists_or_error = orbit_base::FileExists(cache_directory); exists_or_error.has_error()) {
```

Comment:
`orbit_base::FileExists` returns `ErrorMessageOr<bool>`, you need to also check if the bool is false.